maestro-android/src/androidTest/java/dev/mobile/maestro/MaestroDriverService.kt

```diff
@@ -32,10 +33,12 @@ class MaestroDriverService {
             .setWaitForIdleTimeout(0L)
             .setWaitForSelectorTimeout(0L)
 
-        val uiDevice = UiDevice.getInstance(InstrumentationRegistry.getInstrumentation())
+        val instrumentation = InstrumentationRegistry.getInstrumentation()
```

Comment:
This only needs to update server APK right? Not the other one?

Reply:
Yeah, the other one did not have any changes in it since it is just a dummy app